e2e/tests/manualCasePopulation_test.js

```diff
@@ -0,0 +1,78 @@
+
+Feature('Manual case creation');
+
+Scenario('local authority tries to submit after filling mandatory data manually', async (I, caseViewPage, enterOrdersAndDirectionsNeededEventPage, enterHearingNeededEventPage, enterApplicantEventPage, enterChildrenEventPage, enterRespondentsEventPage, enterGroundsForApplicationEventPage, uploadDocumentsEventPage, enterAllocationProposalEventPage) => {
+  await I.logInAndCreateCase(config.swanseaLocalAuthorityUserOne);
+
+  await caseViewPage.goToNewActions(config.applicationActions.enterOrdersAndDirectionsNeeded);
+  enterOrdersAndDirectionsNeededEventPage.checkCareOrder();
+  await I.completeEvent('Save and continue');
+  await caseViewPage.goToNewActions(config.applicationActions.enterHearingNeeded);
+  enterHearingNeededEventPage.enterTimeFrame();
+  await I.completeEvent('Save and continue');
+  await caseViewPage.goToNewActions(config.applicationActions.enterApplicant);
+  enterApplicantEventPage.enterApplicantDetails(applicant);
+  enterApplicantEventPage.enterSolicitorDetails(solicitor);
+  await I.completeEvent('Save and continue');
+  await caseViewPage.goToNewActions(config.applicationActions.enterChildren);
+  await enterChildrenEventPage.enterChildDetails('Timothy', 'Jones', '01', '08', '2015');
+  await I.completeEvent('Save and continue');
+  await caseViewPage.goToNewActions(config.applicationActions.enterRespondents);
+  await enterRespondentsEventPage.enterRespondent(respondents[0]);
+  await I.completeEvent('Save and continue');
+  await caseViewPage.goToNewActions(config.applicationActions.enterGrounds);
+  enterGroundsForApplicationEventPage.enterThresholdCriteriaDetails();
+  await I.completeEvent('Save and continue');
+  await caseViewPage.goToNewActions(config.applicationActions.uploadDocuments);
+  uploadDocumentsEventPage.selectSocialWorkChronologyToFollow();
+  uploadDocumentsEventPage.selectSocialWorkStatementIncludedInSWET();
+  uploadDocumentsEventPage.uploadSocialWorkAssessment(config.testFile);
+  uploadDocumentsEventPage.uploadCarePlan(config.testFile);
+  uploadDocumentsEventPage.uploadSWET(config.testFile);
+  uploadDocumentsEventPage.uploadThresholdDocument(config.testFile);
+  uploadDocumentsEventPage.uploadChecklistDocument(config.testFile);
+  await I.completeEvent('Save and continue');
+  await caseViewPage.goToNewActions(config.applicationActions.enterAllocationProposal);
+  enterAllocationProposalEventPage.selectAllocationProposal('District judge');
+  await I.completeEvent('Save and continue');
+});
+
+
+Scenario('admin sends case to gatekeeper after filling mandatory data manually', async (I, caseViewPage, enterFamilyManCaseNumberEventPage, allocatedJudgeEventPage, sendCaseToGatekeeperEventPage, addHearingBookingDetailsEventPage) => {
+  const caseId = await I.submitNewCaseWithData();
+  await I.navigateToCaseDetailsAs(config.hmctsAdminUser, caseId);
+  await caseViewPage.goToNewActions(config.administrationActions.addFamilyManCaseNumber);
```

Comment:
things like this are already covered in 'Case administration after submission'

Reply:
yup, I missed that, actually all steps are covered in the feature you mentioned, removing it then